.eslintrc.js

```diff
@@ -19,4 +19,13 @@ module.exports = {
       version: 'detect',
     },
   },
+  overrides: [
+    {
+      files: ['add_readmes.js'],
```

Comment:
To match the file name:

```suggestion
      files: ['add-nubots-readmes.js'],
```

Reply:
ah, of course, I was too tired to figure out what was wrong last night haha, thanks